gravitee-am-management-api/gravitee-am-management-api-standalone/gravitee-am-management-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -281,6 +281,8 @@ email:
   from: noreply@my.domain
   username: user@my.domain
   password: password
+  allowlist:
```

Comment:
Not sure about the name, it is not obvious that this entry refer to the allowed `from` values.

If better name can't be found, maybe a comment maybe useful.

Reply:
Also the helm chart will have to be updated to take into account this new setting

Reply:
It's true we have the helm charts in the repo now 👍 
Good catch thanks